kafka-ui-react-app/src/components/ConsumerGroups/Details/Details.tsx

```diff
@@ -54,6 +54,10 @@ const Details: React.FC = () => {
     ? filteredPartitionsByTopic
     : Object.keys(partitionsByTopic);
 
+  const hasAssignedTopics = React.useMemo(() => {
```

Comment:
we don't need the `useMemo` here this is not expensive calculation.
` const hasAssignedTopics = consumerGroup?.data?.topics !== 0`

Reply:
Okay, thank you.
Done